src/config.ts

```diff
@@ -61,6 +64,17 @@ const esSearch = new ElasticSearchService(
 );
 const s3DataService = new S3DataService(dynamoDbDataService, fhirVersion);
 
+const multiTenancyTokenClaim =
+    process.env.MULTI_TENANCY_TOKEN_CLAIM === '[object Object]' || process.env.MULTI_TENANCY_TOKEN_CLAIM === undefined

```

Comment:
Under which conditions `process.env.MULTI_TENANCY_TOKEN_CLAIM === '[object Object]'` can happen? I've never seen anything like this.